src/components/RoomView/RoomView.tsx

```diff
@@ -284,65 +288,62 @@ const RoomView = (props: ClassName) => {
   const claimHost = () => {
     update(hostRef, { playerId, username });
     update(child(playersInRoomRef, playerId), { playerId, username, isHost: true });
-  };
-
-  return (
-    <div className="roomViewBG">
-      <div className={props.className}>
-        <WelcomeBoard />
-
-        <div className="gameStatusClaimHost">
-          {!host && (
-            <p>
-              The host has left, <button onClick={claimHost}>claim host responsibilities</button> to begin game.
-            </p>
-          )}
-          <div className="gameStatus">
-            <GameStatus />
-          </div>
-        </div>
-
-        {/* is there isnt at least one person to each role, setup board should be disabled / not visible 
-      is host AND there is at least one person on each team */}
-        {isHost && (
-          <Popup
-            trigger={
-              <Button style={{ display: 'block', marginLeft: 'auto', marginRight: 'auto' }}> Set Up Board </Button>
-            }
-          >
-            <SetupGame />
-          </Popup>
-        )}
-
-        <div className="flexBox">
-          <TeamOneBox />
-          <div className="boardContainer">
-            {/* player is operative && show operative board, otherwise theyre a spymaster*/}
-            {/* this is working for now, but we probably need more protection to not display 
+  }
+  return ( <div className="roomViewBG">
+    <div className={props.className}>
+      <WelcomeBoard />
+      <div className="gameStatusClaimHost">
+      <GameStatus />
+      <div className='gameStatus'>
+      {!host && 
+      <p>The host has left,  <button onClick={claimHost}>claim host responsibilities</button> to begin game.</p>
+      }</div></div>
+      {/* is there isnt at least one person to each role, setup board should be disabled / not visible */}
+      {/* is host AND there is at least one person on each team */}
+      {isHost && (
```

Comment:
now that i think about it - I don't think this needs to a popup at all - but instead just a component that we render if you are the host && game status is ready, and instead of the close button, when you click the start game button , it does not render.
we can discuss this in class.
maybe the best thing to do for now is to push this onto main as is, and then make a new PR to change it.